### PR TITLE
chore: rename label

### PR DESCRIPTION
**Pull request by ennru**

### CODE REVIEW

File: .github/labeler.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # https://github.com/marketplace/actions/labeler
 
-as-framework:
+kalix-proxy:
```
Comment (octonato) [inline .github/labeler.yml:3]: Sounds strange to have a label `kalix-proxy` in the SDK, no?

Comment (patriknw) [reply .github/labeler.yml:3]: The purpose of this label is to be a "team" label that we use in the aggregated project board. I agree that kalix-proxy might not be the best, but I don't have a better suggestion.

**Comment by patriknw** [comment]
could you also change that label in https://github.com/lightbend/akkaserverless-java-sdk/blob/main/RELEASING.md while you are at it

**Review by octonato — APPROVED** [review]
LGTM

**Pull request merged**